utils/s3/retryable_http_client.cc

```diff
@@ -52,7 +52,9 @@ future<> retryable_http_client::do_retryable_request(http::request req, http::ex
             e = std::current_exception();
             request_ex = aws_exception(aws_error::from_exception_ptr(e));
         }
-
+        if (request_ex.error().get_error_type() == aws::aws_error_type::REQUEST_TIME_TOO_SKEWED) {
+            std::rethrow_exception(e);
```

Comment:
This is wasteful.

Please use `co_await coroutine::return_exception_ptr(std::move(e));` instead.
It would become effective if we get a long burst of such errors, say due to an environmental issue.

Reply:
Done